src/DeploymentProvider/DeploymentHelper.ts

```diff
@@ -315,4 +332,4 @@ export class DeploymentHelper {
         ret["baseUrl"] = testUrl.replace('.test.', '.').replace(regex, '');
         return ret;
     }
-}
\ No newline at end of file
+}
```

Comment:
Here I think I run into some bugs of git or github. I have added newline at the end but it still show no newline here. When going to my PR branch, it still not shows the newline. However when I click the edit button the newline appears and nothing need to commit.
![image](https://user-images.githubusercontent.com/86712051/231950930-a7f4c69c-a698-41fb-8397-d50529c4a2b6.png)